Directory.Packages.props

```diff
@@ -12,7 +13,6 @@
     <PackageVersion Include="FsCheck.Xunit" Version="2.16.6" />
     <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.1" />
     <PackageVersion Include="System.IO.Pipelines" Version="8.0.0" />
-    <PackageVersion Include="Xunit.SkippableFact" Version="1.4.13" />
```

Comment:
Removed and replaced with some code that will just skip depending on which target runtime we support.